llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -3504,6 +3504,11 @@ class VPlan {
     return LiveOuts;
   }
 
+  void removeLiveOut(PHINode *PN) {
```

Comment:
This shouldn't be needed once VPIRInstruction lands (https://github.com/llvm/llvm-project/pull/100735_